lib/dal/tests/integration_test/func/associations.rs

```diff
@@ -41,17 +41,7 @@ async fn for_action(ctx: &mut DalContext) {
 }
 
 #[test]
-async fn for_attribute(ctx: &mut DalContext) {
-    let schema = Schema::find_by_name(ctx, "starfield")
-        .await
-        .expect("could not perform find by name")
-        .expect("no schema found");
-    let schema_variant_id = schema
-        .get_default_schema_variant_id(ctx)
-        .await
-        .expect("could not perform get default schema variant")
-        .expect("default schema variant not found");
-
+async fn for_attribute_with_input_socket_input(ctx: &mut DalContext) {
```

Comment:
The diff didn't make this obvious, but this test only looked at an attribute func with a single input from an `InputSocket`. The new test in this module is similar, but has a `Prop` as the single input.